templates/admin/sources/person/change_form.html

```diff
@@ -0,0 +1,50 @@
+{% extends 'admin/change_form.html' %}
+{% load static %}
+
+{% block content_title %}
+  {% if request.GET.edit %}
+    <h1>Edit {{ opts.verbose_name }}</h1>
+  {% else %}
+    <h1>View {{ opts.verbose_name }}</h1>
+  {% endif %}
+{% endblock %}
+
+{% block object-tools-items %}
+
+  <style>
+    .object-tools a:link.change-link, .object-tools a:visited.change-link {
+      background: #417690;
+    }
+
+    .object-tools .change-link img {
+      margin-right: 3px;
+      padding-bottom: 2px;
+    }
+  </style>
+
+  {{ block.super }}
+  {% if not '?edit=true' in request.get_full_path %}
+    <li>
+      <a href="{% url 'admin:sources_person_change' object_id %}?edit=true" class="change-link"/>
+        <img src="{% static 'img/admin/sources/icon-pencil.svg' %}"  width="11px" height="11px" >
```

Comment:
Here's how it looks for me currently:
<img width="224" alt="Screen Shot 2020-04-17 at 12 10 22 PM" src="https://user-images.githubusercontent.com/327783/79594097-71c8c480-80aa-11ea-9d1c-bbe93bf64cca.png">

What do we think about adjusting like this? cc @jordanebranch 
<img width="253" alt="Screen Shot 2020-04-17 at 12 13 03 PM" src="https://user-images.githubusercontent.com/327783/79594099-72615b00-80aa-11ea-8704-fc1b8abec76c.png">

Reply:
Hey sorry style tag isn't being applied anymore, think I didn't refresh correctly when I deleted the css file.

Reply:
Cool, thanks